### PR TITLE
ISSD-1822 [CP 2.0] Create search template structure for Release Highlights tab

### PR DESCRIPTION
**Pull request by pedroliveirav**
[ISSD-1822](https://liferay.atlassian.net/browse/ISSD-1822)

### CODE REVIEW

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/ddm-templates/release-notes-release-highlights-search/ddm-template.ftl [hunk 1]
```diff
@@ -0,0 +1,284 @@
...
+
+.empty-container {
+	background-color: var(--color-neutral-0);
+	border-color: var(--color-neutral-3);
+	border-style: solid;
+	border-width: 0.0625rem;
+	border-radius: var(--border-radius);
+	padding: 1.5rem 1.5rem 0.0625rem 1.5rem;
+	margin-top: 1.5rem;
+}
```
Comment (samuelleolopes) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/ddm-templates/release-notes-release-highlights-search/ddm-template.ftl:133]: sort in alphabetical order

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/ddm-templates/release-notes-release-highlights-search/ddm-template.ftl [hunk 2]
```diff
@@ -0,0 +1,284 @@
...
+	font-size: 1.125rem;
+	font-weight: 600;
+	line-height: 1.25rem;
+}
+
+.empty-result-message {
+	margin: 5rem auto;
+	max-width: 26.25rem;
+	align-items: center;
+}
```
Comment (samuelleolopes) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/ddm-templates/release-notes-release-highlights-search/ddm-template.ftl:153]: sort in alphabetical order

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/ddm-templates/release-notes-release-highlights-search/ddm-template.ftl [hunk 3]
```diff
@@ -0,0 +1,284 @@
+<style>.pagination-bar {
```
Comment (samuelleolopes) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/ddm-templates/release-notes-release-highlights-search/ddm-template.ftl:1]: should there be a line break?

```
<style>
         .pagination-bar {
```

**Review by samuelleolopes — COMMENTED** [review]
sort in alphabetical order

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/ddm-templates/release-notes-release-highlights-search/ddm-template.ftl (hunk 1 above)
Comment (pedroliveirav) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/ddm-templates/release-notes-release-highlights-search/ddm-template.ftl]: Done.

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/ddm-templates/release-notes-release-highlights-search/ddm-template.ftl (hunk 2 above)
Comment (pedroliveirav) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/ddm-templates/release-notes-release-highlights-search/ddm-template.ftl]: Done.

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/ddm-templates/release-notes-release-highlights-search/ddm-template.ftl [hunk 4]
```diff
@@ -0,0 +1,284 @@
...
+}
+
+.features-header {
+	color: var(--color-neutral-8, #54555F);
+	font-size: 0.813rem;
+	margin-bottom: 0.5rem;
+	padding: 0.25rem 0.25rem 0.25rem 0.625rem;
+}
+
+.features-container {
```
[the remark this answers is not in the document]
Comment (pedroliveirav) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/ddm-templates/release-notes-release-highlights-search/ddm-template.ftl:182]: Done.

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/ddm-templates/release-notes-release-highlights-search/ddm-template.ftl (hunk 3 above)
Comment (pedroliveirav) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/ddm-templates/release-notes-release-highlights-search/ddm-template.ftl]: Done.

**Pull request closed without merging**